homeassistant/components/reolink/host.py

```diff
@@ -379,10 +430,45 @@ def unregister_webhook(self):
         webhook.async_unregister(self._hass, self.webhook_id)
         self.webhook_id = None
 
+    async def _async_long_polling(self, *_) -> None:
+        """Use ONVIF long polling to immediately receive events."""
+        # This task will be cancelled once _async_stop_long_polling is called
+        while True:
+            if self._webhook_reachable:
```

Comment:
with onvif we sleep 0.75 seconds between polls to give a bit of a cooldown when the camera returns no events since sometimes the firmware gets stuck on some cameras and we end up polling in a loop and using all the cpu time otherwise

Reply:
Done, added the 0.75 seconds sleep